skiko/src/macosMain/kotlin/org/jetbrains/skiko/SkiaLayer.macos.kt

```diff
@@ -25,39 +25,75 @@ import platform.Foundation.addObserver
 import platform.darwin.NSObject
 import platform.CoreGraphics.CGRectMake
 
+/**
+ * SkiaLayer implementation for MacOs.
+ * Supports only [GraphicsApi.METAL]
+ */
 actual open class SkiaLayer {
     fun isShowing(): Boolean {
         return true
     }
 
+    /**
+     * [GraphicsApi.METAL] is the only GraphicsApi supported for MacOs.
+     */
     actual var renderApi: GraphicsApi = GraphicsApi.METAL
```

Comment:
Guess we'd better make throwing setter if new value != METAL.

Reply:
added a setter throwing an IAE if new value != METAL.